core/src/main/scala/tech/beshu/ror/accesscontrol/blocks/rules/auth/base/BaseGroupsRule.scala

```diff
@@ -288,6 +290,21 @@ abstract class BaseGroupsRule[GL <: GroupsLogic](val settings: Settings[GL])
 }
 
 object BaseGroupsRule {
-  final case class Settings[GL <: GroupsLogic](permittedGroupsLogic: RuntimeResolvableGroupsLogic[GL],
-                                               usersDefinitions: NonEmptyList[UserDef])
+
+  def name(ruleName: String): RuleName[BaseGroupsRule[GroupsLogic]] = new RuleName[BaseGroupsRule[GroupsLogic]] {
```

Comment:
do we need it? I see one usage and it seems we can get rid of it

Reply:
I removed it from here. But we need to have the RuleName instance for the `RuleDecoder` It is used in the base decoder implementation as the path. So it is move in a slightly changed form to `object ruleDecoders`